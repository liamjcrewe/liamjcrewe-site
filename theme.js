const theme = {
  colors: {
    primary: '#000000',
    secondary: '#ffffff',
    tertiary: '#08c4e6'
  }
}

export default theme
