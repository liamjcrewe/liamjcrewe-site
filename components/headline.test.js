import { render } from '@testing-library/react'
import { ThemeProvider } from 'styled-components'

import 'jest-styled-components'

import theme from '../theme'
import Headline from './headline'

describe('Headline component', () => {
  it('renders and matches snapshot', () => {
    const { container } = render(
      <ThemeProvider theme={theme}>
        <Headline />
      </ThemeProvider>
    )

    expect(container).toMatchSnapshot()
  })
})
