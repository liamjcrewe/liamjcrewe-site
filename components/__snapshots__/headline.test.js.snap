// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline component renders and matches snapshot 1`] = `
.c0 {
  min-height: 20vh;
  text-align: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  font-size: 6rem;
  margin: 0;
  line-height: 1;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c3 {
  background-color: #08c4e6;
  color: #000000;
  padding: 5px 10px;
  border-radius: 15px;
  -webkit-transition: 0.3s;
  transition: 0.3s;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c3:hover {
  background-color: #ffffff;
}

<div>
  <div
    class="c0"
  >
    <div>
      <h1
        class="c1"
      >
        Liam Crewe
      </h1>
      <div
        class="c2"
      >
        <span>
          <a
            class="c3"
            href="mailto:liam.j.crewe@gmail.com"
          >
            Contact:
          </a>
          <span>
             liam.j.crewe@gmail.com
          </span>
        </span>
        <span
          style="padding-right: 5px;"
        >
          Software developer
        </span>
      </div>
    </div>
  </div>
</div>
`;
